.github/CONTRIBUTING.md

```diff
@@ -5,6 +5,7 @@
 * If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](http://www.systers.io).
```

Comment:
In line 2 the link is not working. Can you please look into this

Reply:
![screen shot 2018-08-20 at 9 30 12 pm](https://user-images.githubusercontent.com/17262180/44351770-439f7580-a4c0-11e8-9ab1-a9ffc063dbaa.png)
the link here for slack isn't working either.

Reply:
That's a known issue because systers.io from GitHub pages is not forwarding to the new website deployed on Heroku. That's in progress so that will be fixed in the near future. So I think we don't need to change it because then we would have to change back again probably. What do you think? @m-murad